package.json

```diff
@@ -2,16 +2,16 @@
   "author": "Randy Merrill <Zoramite+github@gmail.com> (http://forthedeveloper.com)",
   "name": "less-middleware",
   "description": "LESS.js middleware for connect.",
-  "version": "3.0.1",
+  "version": "3.9.0",
```

Comment:
Can you change this back. It doesn't need to jump that much.

Reply:
Sure thing -- Sorry, I was assuming that the minor version of the middleware was following the minor version of less.js. Also, this change is somewhat breaking as it drops support for Node <4. 

I'll remove the version change from the commit and defer to the package maintainer on this.